.circleci/config.yml

```diff
@@ -13,6 +13,18 @@ jobs:
           name: Enable Py3
           command: |
             pyenv global 3.7.0
+      - run:
+          name: Run Flake8 Code Lint
```

Comment:
since we have enabled py37 globally, we can move the lint and format check process out of container.